### DISCUSSION TITLE
Using literal_separator option with GlobSet(Builder)

### DISCUSSION THREAD

**Discussion by ruurdadema** [BurntSushi/ripgrep#1771] [Q&A] [answered]
When using `GlobBuilder` we have the option to select whether a literal `/` is required to match using the `literal_separator` builder method.

Is there any way of using this option with `GlobSetBuilder` (and on `GlobSet`) as well? Looking into the code it doesn't seem to be implemented for this case and if I'm correct I hereby kindly request this as feature to be added.

In the meanwhile I will look into the possibility of creating a PR for this, but I'm quite fresh to Rust so I'm not sure if that'll succeed.

**Comment by BurntSushi** [comment, marked as answer]
[`GlobSetBuilder::add`](https://docs.rs/globset/0.4.6/globset/struct.GlobSetBuilder.html#method.add) accepts a `Glob`. Build a `Glob` with `literal_separator` set to whatever you want.

**Comment by ruurdadema** [comment]
Ah, of course! Many thanks for your extremely quick and to the point response!
